### PR TITLE
Quantitative changes to the models

### PR DESCRIPTION
**Pull request by daeglin**

### CODE REVIEW

File: frontend/src/app/game/components/graphs/graphs.component.ts [hunk 1]
```diff
@@ -137,9 +137,9 @@ export class GraphsComponent implements AfterViewInit {
         {
           label: new Date(gs.date),
           value: gs.stats.estimatedResistant.total,
-          tooltipLabel: (value: number) => `Imunní po nemoci: ${formatNumber(value)}`,
+          tooltipLabel: (value: number) => `Krátce po nemoci: ${formatNumber(value)}`,
```
Comment (jnwltr) [inline frontend/src/app/game/components/graphs/graphs.component.ts:140]: We should not be changing this, right?

**Review by jnwltr — CHANGES_REQUESTED** [review]
Please, let me know, if this one is ready.

File: frontend/src/app/game/components/graphs/graphs.component.ts (hunk 1 above)
Comment (daeglin) [reply frontend/src/app/game/components/graphs/graphs.component.ts:140]: I think I already removed this patch from the PR. But let me double check

Comment (daeglin) [reply frontend/src/app/game/components/graphs/graphs.component.ts]: Done

**Review by jnwltr — APPROVED** [review]

**Pull request merged**